packages/contracts/test/unit/HealthChecker.t.sol

```diff
@@ -56,7 +56,7 @@ contract HealthCheckerTest is Test {
         healthChecker.setHealthCheck(address(0));
         vm.expectCall(
             address(healthCheck),
-            abi.encodeCall(healthCheck.check, (address(0), 0, 0, 0, 0, 0))
+            abi.encodeCall(healthCheck.check, (address(0), 0, 0, 0, 0, 0, 0))
```

Comment:
Please add test which check that passed not zero params, use fuzz testing

Reply:
Are you not forgot about this?

Reply:
Nope, that change just doesn't make sense since that tests the empty implementation of IHealthCheck rather than normal HealtchCheck implementation. Could you elaborate on why I should reformat the code for the testing mock?

Reply:
Yes, if you test case `abi.encodeCall(healthCheck.check, (address(0), 1, 2, 3, 4, 5, 6))` and check that all params correctly passed, then we will be sure that there no misplaced variables. They all unit256 and all in one line, it easy to pass one param in incorrect place

Reply:
Yes, if you test case abi.encodeCall(healthCheck.check, (address(0), 1, 2, 3, 4, 5, 6)) and check that all params correctly passed, then we will be sure that there no misplaced variables. They all unit256 and all in one line, it easy to pass one param in incorrect place